test/e2e/v1beta1/scripts/gh-actions/run-e2e-experiment.sh

```diff
@@ -42,9 +43,9 @@ if [ -z "$EXPERIMENT_FILES" ]; then
 fi
 
 for exp_name in "${EXPERIMENT_FILE_ARRAY[@]}"; do
-  echo "Running Experiment from $exp_name file"
+  echo "Running Experiment from $exp_name file with image $TRIAL_IMAGE"
   exp_path=$(find ../../../../../examples/v1beta1 -name "${exp_name}.yaml")
-  python run-e2e-experiment.py --experiment-path "${exp_path}" --namespace default \
+  python run-e2e-experiment.py --experiment-path "${exp_path}" --namespace default --trial-image "${TRIAL_IMAGE}" \
```

Comment:
Since we don't need to run `tune` API for each Katib Experiment, should we just create another file:
`run-e2e-tune-api.py` to run our E2E for `tune` API ?

We don't need to pass exp_path and TRIAL_IMAGE to this file.

WDYT @tenzen-y @johnugeorge @Electronic-Waste ?

Reply:
Yeah, I agree we should just create another file `run-e2e-tune-api.py`. 

But I think we need to pass `TRIAL_IMAGE` to this file, because all the images we use are built locally, including the training image `TRIAL_IMAGE`. If not using this image as Trial image, we need to pull other images to the runner in CI/CD. It may takes more time to do this.

WDYT👀 @andreyvelich @tenzen-y @johnugeorge

Reply:
Oh, I see. Those training images may not be suitable for `tune` function. Maybe we should pull some images like `docker.io/tensorflow/tensorflow:2.13.0`?

Reply:
I think, [by default](https://github.com/kubeflow/katib/blob/master/sdk/python/v1beta1/kubeflow/katib/api/katib_client.py#L169C37-L169C58) we use `docker.io/tensorflow/tensorflow:2.13.0`, so it should be fine.